src/main/docs/guide/aot.adoc

```diff
@@ -0,0 +1,5 @@
+____
+https://micronaut-projects.github.io/micronaut-aot/latest/guide/[Micronaut AOT] is a framework which implements ahead-of-time (AOT) optimizations for Micronaut application and libraries. Those optimizations consist of computing at build time things which would normally be done at runtime]
+____
+
+Micronaut Security offers several ahead-of-time optimizations.
```

Comment:
:o: Maybe clarify the tradeoffs of using AOT: you will save requests at the application startup, which can save a few seconds, at the cost of having a binary which is bound to a specific configuration version.

Reply:
I added a asciidoc warning callout https://github.com/micronaut-projects/micronaut-security/pull/868/commits/e4205987b8d2259f4d4d162c4280a8428f5e2257